service/server.py

```diff
@@ -278,11 +278,31 @@ def format_proprietors(proprietors_data):
         name = proprietor.get('name') or ''
         addresses = proprietor.get('addresses') or []
         formatted_proprietor = {}
-        # TODO: proprietor names have potentially a lot more fields to display
-        if 'forename' in name and 'surname' in name:
-            formatted_proprietor["name"] = name['forename'] + ' ' + name['surname']
+        if 'name_information' in name:
+            formatted_proprietor["name_information"] = ', ' + name['name_information']
+        if 'name_supplimentary' in name:
+            formatted_proprietor["name_supplimentary"] = ', ' + name['name_supplimentary']
+        if 'charity_name' in name:
+            charity_name = ' of '
+            if name['charity_name'].endswith(")"):
+                charity_name += '('
```

Comment:
don't think we should do that? :( if the data is broken then let it be broken unless Ian is happy with us making those modifications; conscious that this should be happening in the migration code instead of in all clients